client/components/deposits-overview/test/index.tsx

```diff
@@ -335,74 +340,80 @@ describe( 'Deposits Overview information', () => {
-			'https://woo.com/document/woopayments/stripe-capital/overview/'
-		);
-	} );
-
-	test( `Doesn't render capital loan notice if deposit does not include financing payout`, () => {
-		const overview = createMockOverview( 'usd', 100, 0, 'estimated' );
-		mockUseDepositIncludesLoan.mockReturnValue( {
-			includesFinancingPayout: false,
-			isLoading: false,
-		} );
-		mockDepositOverviews( [ createMockNewAccountOverview( 'eur' ) ] );
-		mockUseSelectedCurrency.mockReturnValue( {
-			selectedCurrency: 'eur',
-			setSelectedCurrency: mockSetSelectedCurrency,
-		} );
-
-		const { queryByRole, queryByText } = render(
-			<NextDepositDetails isLoading={ false } overview={ overview } />
-		);
-
-		expect(
-			queryByText(
-				'deposit will include funds from your WooCommerce Capital loan',
-				{
-					exact: false,
-					ignore: '.a11y-speak-region',
-				}
-			)
-		).toBeFalsy();
-		expect(
-			queryByRole( 'link', {
-				name: 'Learn more',
-			} )
-		).toBeFalsy();
-	} );
+	// test( 'Renders capital loan notice if deposit includes financing payout', () => {
+	// 	const overview = createMockOverview( 'usd', 100, 0, 'estimated' );
+	// 	mockUseDepositIncludesLoan.mockReturnValue( {
+	// 		includesFinancingPayout: true,
+	// 		isLoading: false,
+	// 	} );
+	// 	mockDepositOverviews( [ createMockNewAccountOverview( 'eur' ) ] );
+	// 	mockUseSelectedCurrency.mockReturnValue( {
+	// 		selectedCurrency: 'eur',
+	// 		setSelectedCurrency: mockSetSelectedCurrency,
+	// 	} );
+
+	// 	const { getByRole, getByText } = render(
+	// 		<NextDepositDetails isLoading={ false } overview={ overview } />
+	// 	);
+
+	// 	getByText(
+	// 		'deposit will include funds from your WooCommerce Capital loan',
+	// 		{
+	// 			exact: false,
+	// 			ignore: '.a11y-speak-region',
+	// 		}
+	// 	);
+	// 	expect(
+	// 		getByRole( 'link', {
+	// 			name: 'Learn more',
+	// 		} )
+	// 	).toHaveAttribute(
+	// 		'href',
+	// 		'https://woo.com/document/woopayments/stripe-capital/overview/'
+	// 	);
+	// } );
+
+	// test( `Doesn't render capital loan notice if deposit does not include financing payout`, () => {
+	// 	const overview = createMockOverview( 'usd', 100, 0, 'estimated' );
+	// 	mockUseDepositIncludesLoan.mockReturnValue( {
+	// 		includesFinancingPayout: false,
+	// 		isLoading: false,
+	// 	} );
+	// 	mockDepositOverviews( [ createMockNewAccountOverview( 'eur' ) ] );
+	// 	mockUseSelectedCurrency.mockReturnValue( {
+	// 		selectedCurrency: 'eur',
+	// 		setSelectedCurrency: mockSetSelectedCurrency,
+	// 	} );
+
+	// 	const { queryByRole, queryByText } = render(
+	// 		<NextDepositDetails isLoading={ false } overview={ overview } />
+	// 	);
+
+	// 	expect(
+	// 		queryByText(
+	// 			'deposit will include funds from your WooCommerce Capital loan',
+	// 			{
+	// 				exact: false,
+	// 				ignore: '.a11y-speak-region',
+	// 			}
+	// 		)
+	// 	).toBeFalsy();
+	// 	expect(
+	// 		queryByRole( 'link', {
+	// 			name: 'Learn more',
+	// 		} )
+	// 	).toBeFalsy();
+	// } );
```

Comment:
Can we skip tests or do we need to comment them out? 
If we have to comment out, leave a note as to why and using a comment block will leave a smaller changeset.
```suggestion
	/**
	 * Capital loans notice temporarily disabled, tests commented out until resolved. See #7689
	test( 'Renders capital loan notice if deposit includes financing payout', () => {
		const overview = createMockOverview( 'usd', 100, 0, 'estimated' );
		mockUseDepositIncludesLoan.mockReturnValue( {
			includesFinancingPayout: true,
			isLoading: false,
		} );
		mockDepositOverviews( [ createMockNewAccountOverview( 'eur' ) ] );
		mockUseSelectedCurrency.mockReturnValue( {
			selectedCurrency: 'eur',
			setSelectedCurrency: mockSetSelectedCurrency,
		} );

		const { getByRole, getByText } = render(
			<NextDepositDetails isLoading={ false } overview={ overview } />
		);

		getByText(
			'deposit will include funds from your WooCommerce Capital loan',
			{
				exact: false,
				ignore: '.a11y-speak-region',
			}
		);
		expect(
			getByRole( 'link', {
				name: 'Learn more',
			} )
		).toHaveAttribute(
			'href',
			'https://woo.com/document/woopayments/stripe-capital/overview/'
		);
	} );

	test( `Doesn't render capital loan notice if deposit does not include financing payout`, () => {
		const overview = createMockOverview( 'usd', 100, 0, 'estimated' );
		mockUseDepositIncludesLoan.mockReturnValue( {
			includesFinancingPayout: false,
			isLoading: false,
		} );
		mockDepositOverviews( [ createMockNewAccountOverview( 'eur' ) ] );
		mockUseSelectedCurrency.mockReturnValue( {
			selectedCurrency: 'eur',
			setSelectedCurrency: mockSetSelectedCurrency,
		} );

		const { queryByRole, queryByText } = render(
			<NextDepositDetails isLoading={ false } overview={ overview } />
		);

		expect(
			queryByText(
				'deposit will include funds from your WooCommerce Capital loan',
				{
					exact: false,
					ignore: '.a11y-speak-region',
				}
			)
		).toBeFalsy();
		expect(
			queryByRole( 'link', {
				name: 'Learn more',
			} )
		).toBeFalsy();
	} );
	*/
```

Reply:
I agree, skip seems ideal for this purpose 😁